packages/lesswrong/components/review/FrontpageReviewWidget.tsx

```diff
@@ -276,7 +265,7 @@ const FrontpageReviewWidget = ({classes, showFrontpageItems=true, reviewYear}: {
       <div>Cast your final votes for the {REVIEW_NAME_IN_SITU}. (Opens {reviewEndDate.clone().format('MMM Do')})</div>
       <ul>
         <li>Look over nominated posts and vote on them</li>
-        <li>Any user registered before {nominationStartDate.format('MMM Do')} can vote in the review</li>
+        <li>Any user registered before {getReviewStart(REVIEW_YEAR).format('MMM Do')} can vote in the review</li>
```

Comment:
Shouldn't this be `nominationStartDate.format(...)` (since we alerady calculate `getReviewStart(REVIEW_YEAR)` above)?